sys/net/crosslayer/ng_netif/ng_netif.c

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @{
+ *
+ * @file
+ */
+
+#include <stdio.h>
+
+#include <errno.h>
+#include "kernel_types.h"
+#include "net/ng_netif.h"
+
+static ng_netif_handler_t if_handler[] = {
```

Comment:
Addresses are the smallest problem… though I'm still not sure how we handle lists of them with `netapi_set`, since an IPV6 interface always has multiple addresses.